symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/Message.java

```diff
@@ -0,0 +1,30 @@
+package com.symphony.bdk.core.service.message.model;
+
+import com.symphony.bdk.gen.api.model.V4Stream;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apiguardian.api.API;
+
+/**
+ * Message model to be used in {@link com.symphony.bdk.core.service.message.MessageService#send(V4Stream, Message)}
+ */
+@Getter
+@Setter
+@API(status = API.Status.EXPERIMENTAL)
+public class Message {
+
+  // The version of the MessageML format
+  private String version;
```

Comment:
when is the version used?

Reply:
This is an optional field in the endpoint : https://developers.symphony.com/restapi/reference#create-message-v4